chainweb.cabal

```diff
@@ -679,30 +678,43 @@ executable transaction-generator
     ghc-options:
         -threaded
         -with-rtsopts=-N
-    hs-source-dirs: examples
+    hs-source-dirs: examples node
```

Comment:
Oh, my below comment is being caused by this line.  What Stuart is saying is correct.  If you remove `node` here, your package will still have access to everything exported from the `node` directory.  This is the case because of the internal build depend below on line 696.

Reply:
I've resolved this.